scripts/.eslintrc.json

```diff
@@ -1,7 +1,13 @@
 {
+  "env": {
+    "node": true
```

Comment:
We should probably use `shared-node-browser` for `env`:
http://eslint.org/docs/user-guide/configuring#specifying-environments

We can discuss the audience of each package on the mailing list.